### PR TITLE
Add a wait after creating LSO yaml

### PR DESCRIPTION
**Pull request by prsurve**
Fixes: #3050 
Signed-off-by: Pratik Surve <prsurve@redhat.com>

### CODE REVIEW

**Review by Avilir — DISMISSED** [review]

**Comment by prsurve** [comment]
Verified locally

**Review by vavuthu — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -995,7 +995,8 @@ def setup_local_storage(storageclass):
             "Creating optional operators CatalogSource and ImageContentSourcePolicy"
         )
         run_cmd(f"oc create -f {optional_operators_yaml.name}")
-
+        # sleep here to start update machineconfigpool status
+        time.sleep(60)
```
Comment (petr-balogh) [inline ocs_ci/deployment/deployment.py:999]: Will be good if you will print the message mentioned in the comment to the log. WDYT?

Comment (prsurve) [reply ocs_ci/deployment/deployment.py:999]: Sure will add a log here

Comment (prsurve) [reply ocs_ci/deployment/deployment.py:1000]: Done

**Review by petr-balogh — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**